assembl/static2/translations/README.md

```diff
@@ -0,0 +1,45 @@
+This folder contains a generated json file for each language excepted for English.
+
+The English messages are maintained manually in
+`js/app/utils/translations.js`
+
+
+# Export messages to po files
+
+To export the messages, meaning converting English messages from
+`js/app/utils/translations.js` to a `locale/assembl-v2.pot` file, run
+
+    npm run i18n:export
+
+This generates an intermediate file static2/messages.json from which we generate the
+pot file using react-intl-po (but we don't use react-intl or
+babel-plugin-react-intl in this project).
+
+The `i18n:export` command takes care of resyncing the existing `locale/assembl-v2/*.po`
+files with the pot file with the msgmerge command.
+To have the msgmerge command on debian/Ubuntu `apt-get install gettext`, and on
+macOS `brew install gettext; brew link gettext`.
```

Comment:
Should likely add these to fabric so auto-installed instances of Assembl are ready to go for this. But good to know.